ci-operator/config/kubernetes/descheduler/e2e.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
name of file is branch, not job

Reply:
I see.  Updated:

- created `master_e2e.json` and `release-3.10_e2e.json` (they are different
  from `master.json` and `release-3.10.json`)
- updated places that reference them
- created a second job for the release-3.10 branch